### DISCUSSION TITLE
Nuclei workflow template output issue

### DISCUSSION THREAD

**Discussion by Lavan08** [projectdiscovery/nuclei#4022] [Q&A] [answered]
I have created an nuclei workflow template and included one template and subtemplate.The template gets executed and subtemplate is executed while running nuclei scans

**Comment by ehsandeep** [comment, marked as answer]
@Lavan08, can you mention/detail the issue?

**Reply by Lavan08** [reply]
@ehsandeep I created below workflow template

id: key-value-sharing-example
info:
  name: Key Value Sharing Example
  author: pdteam
  severity: info

workflows:
  - template: host_extract.yaml
    subtemplates:
    - template: Workflows/login.yaml

While executing this template the host_extract template makes the expected request to the endpoint but the sub_template I have included is not executed .

**Reply by ehsandeep** [reply]
subtemplates will execute only if `host_extract.yaml` gets matched.

**Reply by Lavan08** [reply]
@ehsandeep now workflow working fine, is there any way to reuse cookies for the subtemplates I tried cookie-reuse not able to get expected result

**Reply by ehsandeep** [reply]
@Lavan08 you need to use `cookie-reuse` in both the root template as well as subtemplate, also there is GH ticket to make it as default behavior in future - https://github.com/projectdiscovery/nuclei/issues/3894

**Reply by Lavan08** [reply]
@ehsandeep for the reusage i need to mention cookie-reuse in each templates or I need to mention in the workflow itself
While using in the workflow I got the following error   line 9: field cookie-reuse not found in type workflows.WorkflowTemplate
